### PR TITLE
Add scopes for components

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Add enabled and disabled scopes for components

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:heart:

**Comment by jbrooksuk** [comment]
Odd, I thought I added the `enabled` scope yesterday?

**Comment by jbrooksuk** [comment]
Nope. I've no idea what happened there, thanks @AntoineAugusti!

**Comment by AntoineAugusti** [comment]
Seems you forgot :P I'm having your back ;)

**Comment by jbrooksuk** [comment]
Odd; https://github.com/cachethq/Cachet/pull/1099/files#r43813505 I definitely did it haha. Oh well, thanks <3

File: resources/views/partials/components.blade.php [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <ul class="list-group components">
     @if($component_groups->count() > 0)
     @foreach($component_groups as $componentGroup)
-    @if($componentGroup->components->where('enabled', true)->count() > 0)
+    @if($componentGroup->components->enabled()->count() > 0)
```
Comment (jbrooksuk) [inline resources/views/partials/components.blade.php]: Just pushed a fix that stops us from using the `enabled` scope on a `Collection`.

Comment (AntoineAugusti) [inline resources/views/partials/components.blade.php]: Sorry, missed this one. The API is almost the same between the Builder and Collection, it's confusing

Comment (jbrooksuk) [inline resources/views/partials/components.blade.php]: Don't worry, I tend to do the same thing :yum:

**Pull request merged**